cmd/clusterctl/client/config/providers_client.go

```diff
@@ -55,6 +55,7 @@ const (
 	OCIProviderName            = "oci"
 	OpenStackProviderName      = "openstack"
 	PacketProviderName         = "packet"
+	TinkerbellProviderName     = "tinkerbell"
```

Comment:
This should be changed to the new naming scheme for non-sig / non-community providers:

> For providers not in the kubernetes-sigs org, in order to prevent conflicts the clusterctl name must be prefixed with the provider’s GitHub org name followed by - (see note below).

> The need to add a prefix for providers not in the kubernetes-sigs org applies to all the providers being added to clusterctl‘s pre-defined list of provider starting from January 2024. This rule doesn’t apply retroactively to the existing pre-defined providers, but we reserve the right to reconsider this in the future.
> 
> Please note that the need to add a prefix for providers not in the kubernetes-sigs org does not apply to providers added by changing the [clusterctl configuration](https://main.cluster-api.sigs.k8s.io/clusterctl/configuration).

https://main.cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#adding-a-provider-to-clusterctl